pdns/opensslsigners.cc

```diff
@@ -1828,15 +1824,15 @@ class OpenSSLEDDSADNSCryptoKeyEngine : public DNSCryptoKeyEngine
   size_t d_len{0};
   int d_id{0};
 
-  std::unique_ptr<EVP_PKEY, void(*)(EVP_PKEY*)> d_edkey;
+  std::unique_ptr<EVP_PKEY, void (*)(EVP_PKEY*)> d_edkey;
```

Comment:
```suggestion
  std::unique_ptr<EVP_PKEY, decltype(&EVP_PKEY_free)> d_edkey;
```

Reply:
Just `Key` should work here, actually. Thanks!